docker-compose.yml

```diff
@@ -84,6 +84,7 @@ services:
         PRODUCT_DEF_URL: ${PRODUCT_DEF_URL}
         PS_CONSTANTS_URL: ${PS_CONSTANTS_URL}
         SNIPPET_CREATION: ${SNIPPET_CREATION}
+        TRACKERS_SYNC_TO_JIRA: ${TRACKERS_SYNC_TO_JIRA}
```

Comment:
Should this be added into `osidb-ops` too?

Reply:
As long as the default value of `False` is OK I think it is not needed. Anyway this commit only allows the previously non-existing option to generally set it via the environment but does not account for any specific environment. When we think it should be defined in some we can propose that elsewhere.